CMakeLists.txt

```diff
@@ -16,6 +18,20 @@ endif()
 # * Running cmake with -DFMOD_ENABLED=OFF
 set(FMOD 0) #Acts as an alias to FMOD_FOUND
 option(FMOD_ENABLED "Use the FMOD Ex library in the sound engine" ON)
+set(OPENAL 0) #Acts as an alias to OPENAL_FOUND
+option(OPENAL_ENABLED "Use the OpenAL library in the sound engine" OFF)
+set(TREMOR 0)
+option(TREMOR_ENABLED "Use Tremor instead of libvorbis for OpenAL" OFF)
+
+if(FMOD_ENABLED AND OPENAL_ENABLED)
+  set(FMOD_ENABLED OFF)
+endif()
+
+set(PANDORA 0)
+option(PANDORA_ENABLED "Build forthe OpenPandora" OFF)
```

Comment:
`option(PANDORA_ENABLED "Build forthe OpenPandora" OFF)`

Just a minor typo there. Should be:

`option(PANDORA_ENABLED "Build for the OpenPandora" OFF)`

(Will not impede acceptance)